### PR TITLE
(#1095) Fixes iOS issue with rounded searchbox

### LINKED ISSUE #1095: Search Box Shape Issue After Resizing (ACSF and Prod)
**Status 6/15:** Agree this is a peculiar issue. Happening in both production and ACSF so it is not a bug we are introducing with the release and from that perspective wouldn't hold up the release. We can create a backlog ticket to be addressed after the release, if Anna is open to this/comfortable. Will defer to Anna/federal team. 

**Confirmed 6/15:** Anna comfortable creating a backlog ticket and handling this in a future release and after R2.1. 

**Description:** 
(From Megan) 
Seeing this issue on prod on mobile:
* NCP site - search box always rounded corners
* Cancer.gov & DCEG - search box is rectangle when you first load the site then when you go to any page it changes to rounded corners

Not seeing this on desktop.

(From Anna) Search box shape on apple devices: Minor display issue that seems to be happening in limited cases on the live site. Open the site in safari or chrome on an apple device. On test site it happens every time on initial load. On live site, to trigger the issue, change orientations. When you change, the shape of the search box and alignment of the search button changes, and does not make the intended display. Seeing on Chrome.

Rounded search box edges case 2:
- ipad Air 5th gen
- ipados version 16.5
- safari (I can't figure out the version)

**Ipad Test**

Image

**iPhone 7 Prod - ios 15.7.6**

Image

**Screen record of prod:** 
https://nih.sharepoint.com/:v:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20Release%202/R2.1/Testing/uat-2023-0614-search-box-display-apple-devices.MP4?csf=1&web=1&e=W7XaHQ

### PR DESCRIPTION
**Pull request by dlescarbeau**
Closes #1095

### CODE REVIEW

**Review by karlikpj — APPROVED** [review]

File: testing/ncids-css-testing/.backstop/reference/ncids_usa-pagination_button_hover_0_document_1_tablet.png
Comment (olitharp-nci) [inline testing/ncids-css-testing/.backstop/reference/ncids_usa-pagination_button_hover_0_document_1_tablet.png:1]: any ideas why pagination is diffing?

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Comment by bennettcc** [comment]
Approving on behalf of product - reviewed on 7/18

**Review by olitharp-nci — APPROVED** [review]

**Comment by sarinapadilla** [comment]
I believe this may be an issue with `usa-input` rather than `nci-autocomplete`. When viewing `nci-autocomplete` in Storybook, there is a small border radius, while on Prod the corners are rectangular like `usa-input`. I'm not sure which of these components is causing the rounded search box.

Unfortunately, I can't see the rounded corner issue in Storybook on an iPad, while I can see it on an ODE. It may be that we need to get this fix on an ODE for testing before we can pass this PR.

**Review by andyvanavery — APPROVED** [review]
Passes Product Review

**Pull request merged**